gadopt/time_stepper.py

```diff
@@ -272,8 +267,7 @@ def __init__(
                         u = self.solution_old + self.a[i][j]*self.dt_const*self.k[j]
                     else:
                         u += self.a[i][j]*self.dt_const*self.k[j]
-                self.F.append(self.equation.mass_term(self.test, self.k[i]) -
-                              self.equation.residual(self.test, u, self.solution_old, fields, bnd_conditions))
+                self.F.append(self.equation.mass(self.k[i]) - self.equation.residual(u))
```

Comment:
as above mass -> mass_term?

Reply:
I like the symmetry between `mass` and `residual` here. Is `mass` incorrect?